linux-anvil-cos7-x86_64/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM centos:7
+
+LABEL maintainer="conda-forge <conda-forge@googlegroups.com>"
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# Install basic requirements.
+COPY scripts/yum_clean_all /opt/docker/bin/
+RUN yum update -y && \
+    yum install -y \
+        bzip2 \
+        sudo \
+        tar \
+        which && \
+    /opt/docker/bin/yum_clean_all
+
+# Run common commands
+COPY scripts/run_commands /opt/docker/bin/run_commands
+RUN /opt/docker/bin/run_commands
+
+# Download and cache new compiler packages.
+# Should speedup installation of them on CIs.
+RUN source /opt/conda/etc/profile.d/conda.sh && \
+    conda activate && \
+    conda create -n test --yes --quiet --download-only \
+        defaults::binutils_impl_linux-64 \
+        defaults::binutils_linux-64 \
```

Comment:
Can you change this to `conda-forge::` instead of `defaults::`?

Reply:
Are we building these in conda-forge somewhere?

Reply:
https://github.com/conda-forge/ctng-compiler-activation-feedstock
https://github.com/conda-forge/ctng-compilers-feedstock

Reply:
You might want to update comp7 then too!